platform-sdk/swirlds-benchmarks/src/jmh/java/com/swirlds/benchmark/CryptoBench.java

```diff
@@ -219,12 +230,14 @@ public void transferPrefetch() throws Exception {
 
             // Warm keys in parallel asynchronously
             final VirtualMap<BenchmarkKey, BenchmarkValue> currentMap = virtualMap;
-            for (int thread = 0; thread < numThreads; ++thread) {
-                final int idx = thread;
+            for (int j = 0; j < keys.length; j += KEYS_PER_RECORD) {
```

Comment:
Why not just use key as the loop variable, instead of assigning it?
`for (int key = 0; key < keys.length; key += KEYS_PER_RECORD) {

Reply:
`key` is referenced in the lambda below, so it has to be (effectively) final. That's why it can't be used as a loop variable